Resources/Prototypes/_Funkystation/Entities/Structures/Machines/Computers/computers.yml

```
@@ -0,0 +1,192 @@
+      state: comm_science
+    - map: ["computerLayerKeys"]
+      state: generic_keys
+    - map: [ "enum.WiresVisualLayers.MaintenancePanel" ]
+      state: generic_panel_open
+  - type: AccessReader
+    access: [[ "ResearchDirector" ]]
+  - type: CommunicationsConsole
+    title: comms-console-announcement-title-science
+    color: "#b54ed6"
+    canShuttle: true
+    global: false 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)